internal/ctrlflow/ctrlflow.go

```diff
@@ -169,8 +182,9 @@ func Obfuscate(fset *token.FileSet, ssaPkg *ssa.Package, files []*ast.File, obfR
 		if passes == 0 {
 			fmt.Fprintf(os.Stderr, "control flow obfuscation for %q function has no effect on the resulting binary, to fix this flatten_passes must be greater than zero", ssaFunc)
 		}
+		hardening := newDispatcherHardening(params.StringSlice("flatten_hardening"))
```

Comment:
Define `params.StringSlice("flatten_hardening")` as a variable like you did with the others above